test/support/mock_view_adapter.ex

```diff
@@ -2,4 +2,5 @@ defmodule ExCell.MockViewAdapter do
   @moduledoc false
   def render(cell, template, args), do: [cell, template, args]
   def render_to_string(cell, template, args), do: [cell, template, args]
+  def live_render(conn, cell, args), do: [conn, cell, args]
```

Comment:
I could also make another file `mock_live_view_adapter` with this function, if that is preferred?

Reply:
No need IMO